cypress/integration/chi-custom-elements/label.spec.js

```diff
@@ -0,0 +1,109 @@
+/// <reference types="Cypress" />
+
+describe('Label', function() {
+  beforeEach(() => {
+    cy.visit('tests/custom-elements/label.html');
+  });
+
+  it('Required Label contains asterisk. ', function() {
+    cy.get('[data-cy="test-label-required"]')
+      .find('abbr')
+      .contains('*');
+  });
+
+  it('Optional Label contains string text. ', function() {
+    cy.get('[data-cy="test-label-optional"]')
+      .find('abbr')
+      .contains('(optional)');
+  });
+
+  it('Class xs is present in base. ', function() {
```

Comment:
Since the test function is the same for all the labels on the page please add an array of sizes and make an iteration over the array to abstract the functionality and have clean code.
As a reference example, please check `alert.spec.js` where we use something similar.
```
const alertBannerSelectors = [
    'alert-banner-success-default',
    'alert-banner-center-success-default',
    'alert-banner-dismissible-success-default',
    'alert-banner-titled-success-default',
    'alert-banner-actionable-success-default'
  ];
```

And then

```
beforeEach(() => {
      alertBubbleSelectors.forEach(s => {
        cy.get(`[data-cy=${s}]`, { timeout: 5000 })
          .should('have.class', 'hydrated')
          .children()
          .should('match', 'div.chi-alert.-bubble')
          .as(s);
      });
    });
```

In case of Label, it would be something like:
```const labelSizes = ['xs', 'sm', 'md', 'lg', 'xl'];```

and

labelSizes.forEach(size => {
    /* Test logic goes here */
});

Reply:
@janibolkvadze Just wanted to confirm if this looks good before pushing it: 

`/// <reference types="Cypress" />

describe('Label', function() {
  beforeEach(() => {
    cy.visit('tests/custom-elements/label.html');
  });

  it('Required Label contains asterisk. ', function() {
    cy.get('[data-cy="test-label-required"]')
      .find('abbr')
      .contains('*');
  });

  it('Optional Label contains string text. ', function() {
    cy.get('[data-cy="test-label-optional"]')
      .find('abbr')
      .contains('(optional)');
  });

  const labelSizes = [
    'test-label-base-xs',
    'test-label-base-sm',
    'test-label-base-md',
    'test-label-base-lg',
    'test-label-base-xl',
    'test-label-required-xs',
    'test-label-required-sm',
    'test-label-required-md',
    'test-label-required-lg',
    'test-label-required-xl',
    'test-label-optional-xs',
    'test-label-optional-sm',
    'test-label-optional-md',
    'test-label-optional-lg',
    'test-label-optional-xl'
  ];

    beforeEach(() => {
      labelSizes.forEach(s => {
        cy.get(`[data-cy=${s}]`, { timeout: 5000 })
          .find('label.chi-label')
          .should('have.class', 'chi-label')
          .as(s);
      });
    });
});`

Reply:
@janibolkvadze Just wanted to confirm if this looks good before pushing it:

```

describe('Label', function() {
  beforeEach(() => {
    cy.visit('tests/custom-elements/label.html');
  });
```

```
  it('Required Label contains asterisk. ', function() {
    cy.get('[data-cy="test-label-required"]')
      .find('abbr')
      .contains('*');
  });
```

```
  it('Optional Label contains string text. ', function() {
    cy.get('[data-cy="test-label-optional"]')
      .find('abbr')
      .contains('(optional)');
  });
```
```

  const labelSizes = [
    'test-label-base-xs',
    'test-label-base-sm',
    'test-label-base-md',
    'test-label-base-lg',
    'test-label-base-xl',
    'test-label-required-xs',
    'test-label-required-sm',
    'test-label-required-md',
    'test-label-required-lg',
    'test-label-required-xl',
    'test-label-optional-xs',
    'test-label-optional-sm',
    'test-label-optional-md',
    'test-label-optional-lg',
    'test-label-optional-xl'
  ];

```
```
    beforeEach(() => {
      labelSizes.forEach(s => {
        cy.get(`[data-cy=${s}]`, { timeout: 5000 })
          .find('label.chi-label')
          .should('have.class', 'chi-label')
          .as(s);
      });
    });
});`
```